### PR TITLE
Reproducibility: Support end-to-end pipeline runs for all languages

### PR DESCRIPTION
**Pull request by anthonymirand**
High level summary of changes

- Update flores dataset reference to the new archived location
- Remove UTF-8 encodings for model outputs
- Create RunScripts (training & testing) for all languages (includes necessary Configs)
- Clean translate config files 
- Reference `Data/` where applicable instead of always referring to the symlink `DataLink/`
- Update references from `Data/captioning` to `DataLink/captioning_new` and `Data/ec_finetuning` to `DataLink/ec_finetuning_new`

Results for the local runs for all languages are found below (this PR still does not reproduce paper results 100%):

image

### CODE REVIEW

**Comment by anthonymirand** [comment]
New results for the local runs for all languages:

image

The low resource languages do note a large percent difference from the paper but the small values of BLEU artificially increase the difference.

File: Configs/en-ne_ec.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ TransferH: false # switch to hard gumbel after the model reaches target accuracy
 seed: 1
 mode: emergent_communication
 language_model_lambda: 0.125
-language_model_path: Output/mbart_lm_lr6e-6
+language_model_path: DataLink/../Output/mbart_lm_lr6e-6
```
Comment (shanest) [inline Configs/en-ne_ec.yml:28]: Some places have `./DataLink` and some don't.  Is there a reason not to?  If not, maybe make all of these consistent with `./` (several places)

File: Configs/en2de_translate.yml [hunk 2]
```diff
@@ -6,7 +6,7 @@ num_beams: 5
 translation_filename: de-en.en.val.de
 
 # data config
-source_data_file: de-en.en.val
+source_data_file: Output/en-de_pipeline/de-en.en.val
```
Comment (shanest) [inline Configs/en2de_translate.yml:9]: Same thing about `./`

File: Configs/en-ne_ec.yml (hunk 1 above)
Comment (anthonymirand) [reply Configs/en-ne_ec.yml:28]: No specific reason I had in mind. We can think about this some more but I'll make everything consistent for now!

File: Configs/en2de_translate.yml (hunk 2 above)
Comment (anthonymirand) [reply Configs/en2de_translate.yml:9]: Ack, will make everything consistent!

**Pull request merged**